dev/README.md

```diff
@@ -1,70 +1,61 @@
-apply selected AMPs
-
-## Setting up your development environment
+## 1. Setting up your development environment
 Although it is possible to work on individual github projects, we recommend working on
 the `alfresco-community-repo`, `alfresco-community-share` and `acs-community-packaging`
 in a single Intellij IDEA project. They depend on each other and typically you 
 will want to make changes to all of them if you are changing the repository code.
 
-~~~
+~~~bash
 mkdir work
 cd work
 git clone git@github.com:Alfresco/alfresco-community-repo.git
 git clone git@github.com:Alfresco/alfresco-community-share.git
 git clone git@github.com:Alfresco/acs-community-packaging.git
 ~~~
 
-## Aliases
-There are a set of aliases to help with building. You may find them useful, as they will help you only build selected parts
-of the code base and will save you lots of typing.
+## 2. Aliases
+There are a set of aliases to help with building. You may find them useful, as they will help you only build selected parts of the code base and will save you lots of typing.
 
-Aliases ending in `D` provide Maven commands for building local Docker images. The AMPS environment variable will be of
-interest, if you wish to build AMPs included in the repo and share projects. 
+Aliases ending in **`D`** provide Maven commands for building local Docker images. The AMPS environment variable will be of interest, if you wish to build AMPs included in the repo and share projects. 
 
-The `aliases` file includes a more detailed description.
-~~~
+**You may also build **`alfresco-community-repository`**, **`alfresco-community-share`** and **`alfresco-community-packaging`** with Docker images at once with **`comD`**.**  
+
+The **`aliases`** file includes a more detailed description.
+~~~bash
 source acs-community-packaging/dev/aliases
 ~~~
 
 ## Link the projects
-Generally you will want to link the different projects together by modifying the top level
-pom.xml files of each downstream project so that they reference the SNAPSHOT versions of the
-upstream projects. To help do this see the `acs-community-packaging` project's `scripts/dev/linkPoms.sh` and
-`scripts/dev/unlinkPoms.sh` scripts.
+Generally you will want to link the different projects together by modifying the top level pom.xml files of each downstream project so that they reference the SNAPSHOT versions of the upstream projects. To help do this see the **`acs-community-packaging`** project's **`scripts/dev/linkPoms.sh`** and **`scripts/dev/unlinkPoms.sh`** scripts.
 
-~~~
+~~~bash
 sh acs-community-packaging/scripts/dev/linkPoms.sh
 ~~~
 
-## Build the alfresco-community-repo project
-Build the `alfresco-community-repo` projects (if you have not
-done so already), so that your changes are in the community alfresco.war file.
-~~~
+## 3. Build the Repo project
+Build the `alfresco-community-repo` projects with **`comR`**, so that your changes are in the community alfresco.war file.
```

Comment:
I think this rewording here is unnecessary, `comR` is already mentioned as an alternative in the presented code block that explains how to build the community repo.